src/libostree/ostree-bootloader-zipl.c

```diff
@@ -80,6 +83,203 @@ _ostree_bootloader_zipl_write_config (OstreeBootloader  *bootloader,
   return TRUE;
 }
 
+
+
+static gboolean
+_ostree_secure_execution_get_keys (GArray **keys, GError **error)
+{
+  const char *keyspath = "/etc";
+  g_autoptr(GFile) path          = g_file_new_for_path (keyspath);
+  g_autoptr(GFileEnumerator) etc = g_file_enumerate_children (path, G_FILE_ATTRIBUTE_STANDARD_NAME,
```

Comment:
This is OK as is, though in newer code I've been tending to skip the `GFile*` APIs and use the more raw `glnx_dirfd_*` etc.